docs/design/nab_and_nar_status_update.md

```diff
@@ -124,69 +124,107 @@ It is similar for a NonAdminRestore.
 ```mermaid
 %%{init: {'theme':'neutral'}}%%
 graph
-event1{{predicate: Create event NAB}} == If event is accepted ==>
 
-reconcileStart1[/Reconcile start\] ==>
 
-phaseNew["`status.phase to **New**`"] -. Requeue: false, err .-> reconcileStart1
-phaseNew ==>
+predicateUpdateNabEvent{{predicate: Accepted **update** event NAB}};
+predicateCreateNabEvent{{predicate: Accepted **create** event NAB}};
+predicateUpdateVBEvent{{predicate: Accepted **update** event Velero Backup}};
 
-reconcileExit1[\Requeue: true, nil/]
-reconcileExit1 ==> question(is NonAdminBackup Spec valid?) == yes ==> reconcileStart2
-question == no ==> reconcileStartInvalid1
+requeueTrueNil[\"Requeue: true"/];
+requeueFalseNil[\"Requeue: false"/];
+requeueFalseErr[\"Requeue: false, error"/];
+requeueFalseTerminalErr[\"Requeue: false, Terminal error"/];
 
-reconcileStart2[/Reconcile start\] ==>
+reconcileStartAcceptedPredicate[/Reconcile start\];
 
-conditionsAcceptedTrue["`status.conditions[Accepted] to **True**`"] -. Requeue: false, err .- reconcileStart2
-conditionsAcceptedTrue ==>
+questionPhaseStatusSetToNew{"Is status.phase: **New** ?"};
```

Comment:
I would change all question that are not `success?` to use rectangle like shapes, otherwise they occupy to much space in diagram